File: src/main/java/net/kjp12/commands/utils/GlobalVariables.java
package net.kjp12.commands.utils;

import java.io.PrintWriter;

@Deprecated(forRemoval = true)
public final class GlobalVariables {
    public static final PrintWriter STDOUT_WRITER = new PrintWriter(System.out),
            STDERR_WRITER = new PrintWriter(System.err);

    private GlobalVariables() {
    }
}

File: src/main/java/net/kjp12/commands/utils/GlobalVariables.kt
@file:JvmName("GlobalVariables")

package net.kjp12.commands.utils

import java.io.PrintWriter

@JvmField
val STDOUT_WRITER: PrintWriter = PrintWriter(System.out)
@JvmField
val STDERR_WRITER: PrintWriter = PrintWriter(System.err)